adis/adis.py

```diff
@@ -158,3 +166,22 @@ def dumps(self):
 
     def __repr__(self):
         return """Adis containing %d Adis-files""" % len(self.files)
+
+    @staticmethod
+    def add_string_value(obj: list, entity_data_dict: dict):
+        """ Adds string value to definition list
+
+        Args:
+            obj (list): containing the logical adis files and their contents as builtin types without string values
+            entity_data_dict (dict): dictionary of mapping values for \
+             entity numbers (e.g. {"0080004": "Betriebsnummer"})
+
+        Returns:
+             list: containing the logical adis files and their contents as builtin types with string values
+
+        """
+        for defs in obj[0]:
+            for items in obj[0].get(defs).get("definitions"):
+                x = entity_data_dict.get(items['item_number'][2:], items['item_number'][2:])
+                items['string_value'] = x
```

Comment:
since the definitions already contains the `item_number` I would suggest to rename `string_value` to `item_name` so that one gets immediately that these keys belong together

Reply:
Sorry for my late reply, i am a bit busy now with some certifications.. i will change the pr asap :)